internal/go/skycfg/yaml.go

```diff
@@ -86,10 +86,22 @@ func fnYamlUnmarshal(t *starlark.Thread, fn *starlark.Builtin, args starlark.Tup
 
 // toStarlarkValue is a DFS walk to translate the DAG from go to starlark
 func toStarlarkValue(obj interface{}) (starlark.Value, error) {
+	if obj == nil {
+		return starlark.None, nil
+	}
 	rt := reflect.TypeOf(obj)
+	v := reflect.ValueOf(obj)
 	switch rt.Kind() {
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		return starlark.MakeInt64(v.Int()), nil
+	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
+		return starlark.MakeUint64(v.Uint()), nil
+	case reflect.Bool:
+		return starlark.Bool(v.Bool()), nil
+	case reflect.Float32, reflect.Float64:
```

Comment:
Since skycfg has floats disabled, probably shouldn't be supporting these on the ingest either.

Reply:
The `skycfg` package doesn't have a position on whether floats are enabled or not. This is controlled by a global mutable variable in the `starlark` package.

Reply:
Fair enough. Should this test for `resolve.AllowFloat` though so that floats don't accidentally leak into configs even though they are disabled?

Reply:
That sounds reasonable, but it would probably need to be at a lower level since floats can also arrive via JSON or Protobuf parsing.